src/ralph/export_to_ng/management/commands/export_model.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+POSSIBLE_MODELS = """
+Models independent:
+AssetLastHostname
++Environment
++Service
+Manufacture
++Category
+ComponentModel
+Warehouse
+DataCenter
+Accessory
+Database
+VIP
+VirtualServer
+CloudProject
+LicenceType
+SoftwareCategory ile
+SupportType
+
+Custom resoruces models:
++ralph.assets.models.assets.AssetModel
+ralph.assets.models.components.GenericComponent
+ralph.back_office.models.BackOfficeAsset
+ralph.data_center.models.physical.ServerRoom
+ralph.data_center.models.physical.Rack
++ralph.data_center.models.physical.DataCenterAsset
+ralph.data_center.models.physical.Connection
+ralph.data_center.models.components.DiskShare
+ralph.data_center.models.components.DiskShareMount
+ralph.licences.models.Licence
+ralph.supports.models.Support
+
+ManyToMany Models:
++ralph.assets.models.assets.ServiceEnvironment
+    - service,
+    - environment
+ralph.licences.models.LicenceAsset
+    - licence
+    - asset
+ralph.licences.models.LicenceUser
+    - licence
+    - user
+ralph.data_center.models.physical.RackAccessory
+    - accessory
+    - rack
+"""
+
+import os
+from optparse import make_option
+from import_export import resources
+
+from django.core.management.base import BaseCommand
+from django.db.models import get_models
+
+import ralph_assets
+from ralph.export_to_ng import resources as ralph_resources
+from ralph.discovery import models_device
+
+
+APP_MODELS = {model._meta.object_name: model for model in get_models()}
+APP_MODELS.update({
+    # exceptions for ambigious models like Warehouse,
+    # which is in Scrooge and in
+    # Assets, we need only asset's one so code below:
+    'Warehouse': ralph_assets.models.Warehouse,
+    'Service': models_device.ServiceCatalog,
+})
+
+
+def get_resource(model_name):
+    """Return resource for import model."""
+    resource_name = model_name + 'Resource'
+    resource = getattr(ralph_resources, resource_name, None)
+    if not resource:
+        model_class = APP_MODELS[model_name]
+        resource = resources.modelresource_factory(model=model_class)
+    return resource()
+
+
+class Command(BaseCommand):
+
+    EXPORT_LIMIT = 10000
+
+    help = os.linesep.join([
+        'Export data in Ralph-NG format.',
+        os.linesep,
+        'Possible models to export:',
+        POSSIBLE_MODELS,
+    ])
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--model_name',
+            help=(
+                'The model name up to export '
+                '(type "ralph export_to_ng -h" for possible models)'
```

Comment:
on the one hand yes, but on the other side export format is perfectly tailored to ng (export format does not suit current ralph models).